zephyr/module.yml

```diff
@@ -36,9 +36,9 @@ blobs:
     description: "Binary libraries supporting Telink TL321X series BLE subsystems"
 
   - path: lib_zephyr_tl721x.a
-    sha256: ab0fcb214922e33b4a397748051d9fccae99ce65775ecd13af2b6aec772633c5
+    sha256: 9c6af7a45de2e0bf99ad116d6a39ddc498e4dfccd81e2eef0369965f379d9e73
     type: lib
     version: 'V4.0.4.2-WIP'
     license-path: tlsr9/ble/license.txt
-    url: http://wiki.telink-semi.cn/wiki/protocols/Zephyr/binaries/public/tl721x_lib/lib_zephyr_tl721x_cc4a84ab75b5797aee031777329559fca067ff4d.a
+    url: http://wiki.telink-semi.cn/wiki/protocols/Zephyr/binaries/public/tl721x_lib/lib_zephyr_tl721x_404e51e546195e114e4780b69956fe43f635c4f5.a
```

Comment:
This hash 404e51e546195e114e4780b69956fe43f635c4f5 is from [tl721x_bringup](http://192.168.48.36/src/open_platform/telink_b9x_ble_src/tree/tl721x_bringup) branch.
Let's merge lib changes into [master_tl721x](http://192.168.48.36/src/open_platform/telink_b9x_ble_src/tree/master_tl721x) branch before mergind HAL into develop

Reply:
@s07641069 
The master_tl721x has been created and also set to a protected breach and the library file has been rebuilt and uploaded.